quickstart/examples/transformer/README.md

```diff
@@ -0,0 +1,73 @@
+### DeDuplicationTransformer
+* Transformation Class:
+    * `io.odpf.dagger.functions.transformers.DeDuplicationTransformer`
+* Contract:
+    * After Selecting columns by SQL, you need to reselect the desired columns with the help of an internal source. Following transformation arguments can be passed:
+        * `key_column`: This value will be used as the deduplication key (other events with the same key will be stopped).
+        * `ttl_in_seconds`: The TTL configuration will decide how long to keep the keys in memory. Once the keys are cleared from memory the data with the same keys will be sent again.
+* Functionality:
+    * Allows deduplication of data produced by the dagger i.e records with the same key will not be sent again till the TTL expires.
+    * Can be used both on `post-processor` and `pre-processor`
+* Example:
+    * SQL:
+      ```
+      SELECT
+        data1,
+        data2
+      FROM
+        data_stream
+      ```
+    * POST PROCESSOR CONFIG:
+      ```
+      {
+        "internal_source": [
+          {
+            "output_field": "data1",
+            "value": "data1",
+            "type": "sql"
+          },
+          {
+            "output_field": "data2",
+            "value": "data2",
+            "type": "sql"
+          }
+        ],
+        "transformers": [
+          {
+            "transformation_arguments": {
+              "key_column": "data1",
+              "ttl_in_seconds": "3600"
+            },
+            "transformation_class": "io.odpf.dagger.functions.transformers.DeDuplicationTransformer"
+          }
+        ]
+      }
+      ```
```

Comment:
Same as this : https://github.com/odpf/dagger/pull/214/files#r1049652605

Let's not recommend best practices or configs here. We only need to help the user run a demo example.

We can explain these things in the future story, where we will create a dedicated folder called `Concepts`

Reply:
done